post.hbs

```diff
@@ -64,32 +64,24 @@
             </section> }}
             </footer>
         </article>
-
-        <aside class="gh-readmore">
-            <div class="gh-readmore-inner w-full max-w-screen-xl mx-auto my-0">
-                <div class="gh-readmore-next">
-                    {{#next_post}}
-                        <a href="{{url}}">
-                            <i class="fa fa-chevron-left" aria-hidden="true"></i>
-                            &nbsp;
-                            <h4>
-                                {{title}}
-                            </h4>
-                        </a>
-                    {{/next_post}}
-                </div>
-                <div class="gh-readmore-prev">
-                    {{#prev_post}}
-                        <a href="{{url}}">
-                            <h4>
-                                {{title}}
-                            </h4>
-                            &nbsp;
-                            <i class="fa fa-chevron-right" aria-hidden="true"></i>
-                        </a>
-                    {{/prev_post}}
-                </div>
-            </div>
+        
+        {{!-- implementation of related posts --}}
+        <article>
+            <br> {{!-- spacing --}}
+            <hr> {{!-- divider --}}
+            <br> 
```

Comment:
Rather than using `<br>` inbetween dividers and the other element, I think it would be best to style them accordingly.

Reply:
I repurposed the readmore.css to style these elements and I hate developing css on this setup cos you gotta restart ghost each damn time